main.go

```diff
@@ -57,16 +61,24 @@ func main() {
 		exitf("Failed to connect to Spanner: %v", err)
 	}
 
-	stdin, err := readStdin()
+	input, err := readStdin()
 	if err != nil {
 		exitf("Read from stdin failed: %v", err)
 	}
 
-	var exitCode int
 	if opts.Execute != "" {
-		exitCode = cli.RunBatch(opts.Execute, opts.Table)
-	} else if stdin != "" {
-		exitCode = cli.RunBatch(stdin, opts.Table)
+		input = opts.Execute
+	} else if opts.File != "" && opts.File != "-" {
```

Comment:
I think this works for when user gives input from pipe,
```
$ <INPUT> | spanner-cli -f=-
```
but otherwise it enters interactive mode.

If we allow `-f=-` for stdin, I would expect spanner-cli blocks execution until the tool finished reading stdin.
```
$ spanner-cli -f=- # block
SELECT * FROM t1; # start input
<Ctr-D> # finish input
<Show results>
```

Reply:
It should be fixed in https://github.com/cloudspannerecosystem/spanner-cli/pull/27/commits/755e4a8408b10970745639a3cce3ced54ffe0b9b